mix.exs

```diff
@@ -1,13 +1,13 @@
 defmodule OpenIDConnect.Mixfile do
   use Mix.Project
 
-  @version "0.2.2"
+  @version "1.0.0"
 
   def project do
     [
       app: :openid_connect,
       version: @version,
-      elixir: "~> 1.3",
+      elixir: "~> 1.13",
```

Comment:
is there anything code-wise that required that required the bump in Elixir?

Reply:
Finch requires it to be 1.7, but above that - I'm not sure. It's not easy to track, so I just updated it to the version used in our production. "Better safe than sorry".

Reply:
ok that's fine then, wanted to make sure it was at the broadest possible range

Reply:
`Enum.sort/2` that I've used is introduced in Elixir 1.10.

Reply:
let's go with 1.10 then, there are sometimes reasons why consumers cannot bump elixir versions but we should aim for least common denominator

Reply:
Sounds good, I will setup CI to test against different OTP and Elixir versions.

Reply:
@bcardarella the minimum required Elixir version will be 1.12 because of `URI.encode_query/1` usage. I also added a new line at the end of LICENSE.md to make my editor and static analysis happy (most editors will even do that for you automatically).

For a successful CI job see: https://github.com/firezone/openid_connect/actions/runs/4768193206/jobs/8477240076